desktop/components/artwork_masonry/view.coffee

```diff
@@ -19,7 +17,7 @@ module.exports = class ArtworkMasonryView extends Backbone.View
     @subViews.push artworks.map ({ id }) =>
       $el = @$(".js-artwork-brick[data-id='#{id}']")
 
-      view = new BrickView[$el.data('type') or 'artwork']
```

Comment:
Beforehand this was selecting the `artwork_brick` class, or the `auction_artwork_brick` class which is deprecated. (That reminds me, I should add a note.)